src/collections/_documentation/workflow/user-settings.md

```diff
@@ -0,0 +1,187 @@
+
+Control workflow notifications (for example, in issue assignment, resolution status, and comments) by clicking a radio button under Send Me Workflow Notifications. You can also fine-tune workflow notifications per project.
+
+For more details, see the [full documentation on Alerts & Notifications]({%- link _documentation/workflow/alerts-notifications/index.md -%})
+
+## Email Routing
+
+Route email notifications to an alternative email address and keep your project notifications organized. 
+
+{% asset user-settings/us-eroute.png alt="Email Routing field in Notifications under User Settings" width="450" %}
+
+## Weekly Reports
+
+Sentry generates reports per project and sends them once a week. You can fine-tune your reports per project.
+
+{% asset user-settings/us-wrfield.png alt="Weekly Reports field in Notifications under User Settings" width="450" %}
+
+The following report sample shows details on Events Seen This Week, Events by Issue Type, Events by Project, and activity over time.
+
+{% asset user-settings/us-ereport.png alt="Weekly Report email sample" width="745" %}
+
+## Deploy Notifications
+
+Choose a Deploy Notifications option to keep track of deployment details like release version, environments, and commit reviews. You can fine-tune options from the dropdown next to each organization.
+
+{% asset user-settings/us-deploy.png alt="Deploy Notifications selection in Notifications under User Settings" width="673" %}
+
+## My Activity
+
+Updates regarding your user account actions, such as claiming unassigned issues, can be turned into an email notification in this section.
+
+{% asset user-settings/us-myact.png alt="My Activity selection in Notifications under User Settings" width="450" %}
+
+{% capture __alert_content -%}
+You can find a shortcut to the Emails panel at the bottom of this section.
+
+{% asset user-settings/us-npanellink.png alt="A link to the Email panel in Notifications under My Activity" width="465" %}
+
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+%}
+
+## Emails
+
+The email address used to login to your Sentry account is by default your primary email address. You can add an alternative email address under the Add Secondary Emails section.
+
+{% asset user-settings/us-eprime.png alt="Primary email field in Emails under User Settings" width="494" %}
+
+{% capture __alert_content -%}
+
+You can find a shortcut to the Notifications panel at the bottom of this section.
+
+{% asset user-settings/us-epanellink.png alt="A link to the Notifications panel in Emails under Additional Email" width="450" %} 
+
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+%}
+
+## Subscriptions
+
+As part of our compliance with the EU’s General Data Protection Regulation (GDPR), you wills only receive marketing campaign emails after explicitly opting-in to any of the listed categories. 
```

Comment:
..."you receive marketing campaign emails only after explicitly opting in to any of the listed categories."